products/ubuntu2004/product.yml

```diff
@@ -46,6 +46,7 @@ platform_package_overrides:
   pam: libpam-runtime
   shadow: login
   sssd: sssd-common
+  systemd-journal-upload: systemd-journal-remote
```

Comment:
I'm not sure if this change is needed.
- `systemd-journal-upload` is the name of the service, not a package that would need overriding. I also don't think the package needs overriding.
- this PR is about Ubuntu 24.04, and this change is on Ubuntu 20.04 product

Reply:
Removed